07-On demand scan as an API/README.md

```diff
@@ -0,0 +1,203 @@
+
+
+Command to generate the token:
+``` PowerShell
+
+$token = Get-MsalToken -TenantId '<tenant-id>' -ClientId '<client-app-id>' -RedirectUri 'https://localhost' -Scopes '<WebAPI-scope>'
+
+```
+
+[Back to top…](README.md#On-this-page)
+
+## API to scan a subscription
+
+Use below PowerShell command to get control scan result.
+
+``` PowerShell
+
+$header = "Bearer " + $token.AccessToken
+$headers = @{"Authorization"=$header;"Content-Type"="application/json";}
+$requestBody = @{"SubscriptionIDList"=@("subscription_id");}
+
+# Example:
+$requestBody = @{"SubscriptionIDList"=@("sub1","sub2");}
+
+
+$method = [Microsoft.PowerShell.Commands.WebRequestMethod]::POST
+$apiUri = "https://azsk-ats-api-ringn.azurewebsites.net/adhocscan/RequestScan"
+
+$apiResponse = Invoke-WebRequest -Method $method -Uri $ascUri -Headers $headers -Body ($requestBody | ConvertTo-Json) -UseBasicParsing
+
+$response = ConvertFrom-Json $apiResponse.Content
+$response
+
+```
+
+**Request body parameter details:**
+|Param Name|Description|Required?
+|----|----|----|
+| SubscriptionId| List of subscription id(s) for scan. | Yes|
+
+</br>
+
+[Back to top…](README.md#On-this-page)
+
+## API to get control scan result
+Use below PowerShell command to get control scan result.
+
+``` PowerShell
+
+$header = "Bearer " + $token.AccessToken
+$headers = @{"Authorization"=$header;"Content-Type"="application/json";}
+$requestBody = @{}
+
+# Example:
+$requestBody = @{"scanRequestId"="scan_request_id";"ControlIdList"=@("control_id1","control_id1");"ResourceNameList"=@("resource1","resource2");}
+
+
+$method = [Microsoft.PowerShell.Commands.WebRequestMethod]::POST
+$subId = <sub-id>
+$apiUri = "https://azsk-ats-api-ringn.azurewebsites.net/adhocscan/$subId/<sub-id>/ControlScanResult"
+
+$apiResponse = Invoke-WebRequest -Method $method -Uri $ascUri -Headers $headers -Body ($requestBody | ConvertTo-Json) -UseBasicParsing
+
+$response = ConvertFrom-Json $apiResponse.Content
+
+$folderPath = [Environment]::GetFolderPath("MyDocuments") 
+if (Test-Path -Path $folderPath)
+{
+    $folderPath += "\AzTS\Subscriptions\$($subId.replace("-","_"))\$((Get-Date).ToString('yyyyMMdd_hhmm'))\"
+    New-Item -ItemType Directory -Path $folderPath | Out-Null
+}
+
+if (($response | Measure-Object).Count -gt 0)
+{
+    $response | ConvertTo-json | out-file "$($folderpath)\ControlScanResult.json"   
+    Write-Host "Check control scan result for requested subscription at[$($folderPath)]..." -ForegroundColor Green    
+}
+
+```
+
+
+**Request body parameter details:**
+|Param Name|Description|Required?
+|----|----|----|
+| ScanRequestId | To get control scan result with respect to the scan request id.| Yes |
+| ControlIdList| List of control ids to get control result only for specific controls.| No |
+| ResourceNameList | List of resources to get control result only for certain resources.| No |
+
+
+</br>
+
+> **Note:** 
+> </br>
+> _1. Enable the flag 'OnDemandScanAPI' in order to expose API endpoints._
```

Comment:
This is a note for admin. This should have a separate section.

Reply:
Added as part of pre-requisites.